src/io.c

```diff
@@ -135,7 +135,7 @@ avifIO * avifIOCreateFilePtrReader(FILE * f, avifBool closeOnDestroy)
     avifIOFileReader * reader = avifAlloc(sizeof(avifIOFileReader));
     memset(reader, 0, sizeof(avifIOFileReader));
     reader->f = f;
-    reader->closeOnDestroy = closeOnDestroy;
+    reader->closeOnDestroy = AVIF_TRUE;
```

Comment:
Could you also delete the closeOnDestroy field of struct avifIOFileReader at line 73? Thanks.

Reply:
Good catch. Done.